openfecwebapp/config.py

```diff
@@ -5,7 +5,7 @@
 api_location_public = os.getenv('FEC_WEB_API_URL_PUBLIC', api_location)
 api_version = os.getenv('FEC_WEB_API_VERSION', 'v1')
 host = os.getenv('FEC_WEB_HOST', '0.0.0.0')
-port = os.getenv('VCAP_APP_PORT', '3000')
+port = os.getenv('FEC_WEB_PORT', '3000')
```

Comment:
It turns out that this code never gets touched on CF, which is confusing. This is only used when we hit the `if __name__ == '__main__'` block in `__init__.py`, which doesn't happen when running the app with gunicorn.

In other words, whenever `VCAP_APP_PORT` is defined on the environment, this variable isn't used. Unless we want to use it for local development, which I haven't been doing so far.